apps/web/src/components/forms/signin.tsx

```diff
@@ -39,6 +51,31 @@ export const SignInForm = ({ className }: SignInFormProps) => {
     resolver: zodResolver(ZSignInFormSchema),
   });
 
+  const timer = useRef<NodeJS.Timeout | null>(null);
+
+  useEffect(() => {
+    const error = searchParams?.get('error');
+    if (error) {
+      timer.current = setTimeout(() => {
+        // FIXME: Toast not firing without the TimeOut
```

Comment:
@Mythie This is what I talked about. @G3root solved it with a timeout, take it out and the toast fails to run but if you place a console log inside the useEffect, it runs on the page.

Reply:
Gotcha, that was an annoying one. I tidied it a little but it's good to go!